components/brave_wallet_ui/assets/svg-icons/nft-empty-state.svg

```diff
@@ -0,0 +1 @@
mBd0Y/CTAAAgAElEQVSPxw8cOnTomre97W3PJ1+KT1JFiY5XbW1txZfYWaooMlNbt25NiFIasanKebNLn5jHcaNNqfJa+1Uj1RHcKDivuaEQLHBiCWh0dXU5tbW1s2IvyHj27Nnzl5WVlV9PB1jgGLu7++C1bW9II7UzfmdPYQfdGJGZOJEyKLcgIpQ+HguC7MP9hyCCBzUhE5mMQ3/y58hUIpmbd3rfySQQwaOP8MS8anQvC/8zeOzYsc+tX7/+VyL3WeyCysitY+AmQ+GpolRnW6TTyG0EFqI3EKKWkqmhWBUUCszEG4p3+JHXOAuaWezevftj1dXVtzuOszD89XhyCxitvfONPdDXN5iO5udMm27eLOKhlUx9YpkIf3p5IDIwAyKo0koADSLs62uqmEhlbg6cW5kPiwpSk2W3t7f3xsbGxgeDPtdC9JCp7BZos8BEgvQRq1kJFqzdIug4C53cUDgJXryhaLDYs2fPxysrK7+WLrBAPTQauffsPXBSKuI5s5OncSBoAyGqLAw2xN8tiIhABNlHko0kmci0LSSRtIkkYIxiImGDyKL8XPd41dJc7XdaXyttaGjon1etWoXOLtzAPJZZoDdUfX291G6hE2shSlWO7XHyQtFjzC411KyeY1rm22/nzqwps9ABC9NEgqxxG/u+d+/eT1RUVPyD4ziGBzP7Wpezbh4cHIadb+yFgYGh4Cq1NUklgN5YCByVFBshb8ypenPOlinCLAM0iEwmkuAxSWwiCRiPn1BnBQEiuImcVZ4P60vDP4ebzMPo6Oj3VqxYcZPqLG5SXicwD8vqJBQUAQabIyormQVvoQfpDWXiNkszDPxdZeCm4yz27t37yYqKiq+mEyzceIvDx2H3nrZQc0Nly+aUzn4WFyOITKeBn2Yipcmzwafz8tjPCQngCYQuiCRZyGT8BHhMe2glYMIDiKCUm0ryYGNFvnuIVKo+4+Pjv6irq/sUtqcTayFKVY73B5VQkE31kTVgkRTiSXOnY7PAm0xcZ3XAggYJ2eFHJOusgFl8KgkWNalalLx2MD15W3sHtLV3omotnV2xbXMkgBHqCBwugLhspAwwCFH0TMxnIU6DCBrVp1Va0fgJozpRa+F3MyZ1CoTLIzmwsigX1pTkQXEKvKDoeZqcnPzto48++ukbbrihG79n80PR3+HvqQALbIdmF/g3lY8vc9VQtOsW+5aVDptFEGqo1tbWq8vLy7/sOE5awQIXAQbp7W875KYpx9/tJ/MlgLmyCAMhIGIPpuLPGwsiyDrGpgAGpqa9tOqLcqEyL8cFiRTjhNvheDy+dceOHZ+99NJLd5MRsIDB84YiwEHO4qaZBf7uRw1FwIJnIyYMKFVPiTavpl1nRR0X2SvIYILMOEuzCvb35MSfdJ4Fj1m0trZeU1FRcRMApB0sCGB0HT4G7e2dMDo2NifSgKRqMWdSO246eLSJJFVZ+HtexB5MlUlzxPYlHo+3tbW1XXveeee5sRaESeTm5s68wROwIKyCZhgELFAFhd6bKlWUTo4oHrNIF7vwBBZEkCZnWeA9QamhsC46gtsPWOzfv/+60tLSLzmOU50pCxltGOhKe6jjCBzv7gM8GMl+5oYEMB088cqaBpNSyMubNuJam0h65ziRSAwcP378uvXr1/+avLXLmAUNFAgeouA8HWaBdclSldNnCRE1FOljqqRmBBb0YmbZhcq4nUlgkURmd+ytra03VFRU3AgAGQMWZPJRFXX0WA8cOXIcensHIDZF5XJI1Qqx7aREApgOfjaIlIE93TAlop/VSH9//xfWrFnzkCjWghfFTUDDjxpKBhZsQsF0GbmNwIJ++6HBImh7BbajY+Bmjdr037KT8mhvqLa2ts+XlJTc6DhOVeqXplmLyDgGB0dgaHgEhoaGYWho1D2nO2qZh5kgs6g0qq5c76xkjAh9MJVlIsFP5ODg4D+tXr0as1BrxVqkwn2WTShIjzqVCQWNwIJlFjR4qJiFiQpKBRZ4PSg1VFtb2xdLS0u/AAAZDxZkkSA4s//GJ6IucND/xrLojIzgH/u5XWNubu4ME0FVlj2YKpj5Hhsbe2D58uVfplU8ugkFvTALzGKhiuRmvaHofSBjwYIGB3ZqVOzCL1hgeyTlhyyJIOmX7hncbW1tf19SUnKD4ziVwSy39NWCZ2KwIIJuuAgggwgkLiMZSdvxremTzPxpORfTwSdZCHH1tQdT6c//+Pj4z+rq6j4jAguihuro6MBM14GdmCeL4uYxi3ScxW3ELGRg4ZdZ8A42FyUSJGCB/UHVE6uOwu8N1FA3lZSUfG4ugIXskUAQiaNvO8VKCHgQNoKAgqou+5l7EsDNhdhEyE97MNXJ8xyNRp/55S9/+ZkvfOELPXj1+PHjCUxCyvOIwus6sRZYTsfIrZv2g2YWNKiFvWq1wILW8YtiLfyCBTtQk4yzBDTYOghgyILy2travlxSUnKd4zgVYQs7E+unwYOAydjY+DQTodhINGo9sjJx/vz2CePhpnNnETffMpjPB1NNTU29umPHjmsuvfTSPewhSOQsbtNDkIIAC2QX7Mt6qoPzPIMFu0jDAAv6eEI666xfNRQNfu3t7V8pLi6+Zr6ChWiz4dlFJiYmZwEIAgoCi/3MTQmQVPCVldMR66Ulc/9gqkQisW/fvn3XnX/++S+QWZXZLGTsQhVngfeq7BWkD6Io7oxkFirPizDAgn4EeWBBMwqv6T4OHjx4S2Fh4Wcdxymfm4988KNi1VmYOXeGiVCMJPiWbY2ZIAH3TJFkjAj+LCudPphKtUdkQt9VfUgkEn14rsWpp576n7TxOC8vb5Z+lmUXshPzZMxCBzBuu+02LMZN7ZH0Sk1J2g8jZsFL8UG+CxIsWLdZlJQJWIgSCWI9bBR3e3v714qLiz9twUL1GKmvEzZCgwme10Eb19Hld2rK2kXU0sy+EtVVFbB2Tb3r6oveWtn66enp+XxTU9O/m7jPsmCBY6cPQtKxWeA9suyz
```

Comment:
If passed through SVGO some HREF are removed. One is not removed since we are using a PNG embedded in base64 in a SVG. Do we want to have non-scalable vector graphics in a scalable vector graphic? Cc @fmarier

Reply:
I can't think of a security issue behind this, but it does seem wasteful to encode a PNG into base64 and then wrap it into an SVG. So it seems more of a file-size / bloat concern.

Nice catch for that tool though :)

Reply:
I just took a better look at this and it seems like that SVG contains 3 different PNGs. So it's not just a PNG wrapped inside an SVG.

Reply:
This is the typical false positive, but this strategy cannot identify which url (outside ping back for example, or script included) is contained and, out of precaution it marks all the urls.